app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -419,15 +433,18 @@ class OnboardingFragmentTest {
   @Test
   fun testOnboardingFragment_checkSlide2_getStartedButtonIsNotVisible() {
     launch(OnboardingActivity::class.java).use {
-      onView(withId(R.id.onboarding_slide_view_pager)).perform(scrollToPage(2))
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
       onView(withId(R.id.get_started_button)).check(matches(not(isDisplayed())))
     }
```

Comment:
This test case is failing. I suspect that it isn't correct because the error thrown is - 
```
androidx.test.espresso.NoMatchingViewException: No views in hierarchy found matching: with id: org.oppia.android:id/get_started_button
```
which makes sense. Because there is no `Get Started` button on the 2nd slide. I think we should instead check for the correct exception thrown.